bin/api-docs/gen-components-docs/markdown/index.mjs

```diff
@@ -10,8 +10,8 @@ import { generateMarkdownPropsJson } from './props.mjs';
 
 export function generateMarkdownDocs( { typeDocs, subcomponentTypeDocs } ) {
 	const mainDocsJson = [
-		'<!-- This file is generated automatically and cannot be edited directly. -->\n',
 		{ h1: typeDocs.displayName },
+		'<!-- This file is generated automatically and cannot be edited directly. Make edits via TypeScript types and TSDocs. -->',
```

Comment:
I also appended a note on how to make edits. Especially because there is an "edit link" at the bottom of the Block Editor Handbook pages that [points](https://github.com/WordPress/gutenberg/edit/trunk/packages/components/src/alignment-matrix-control/README.md) to the readme file.

<img width="484" alt="Edit link that says 'Improve it on GitHub'" src="https://github.com/user-attachments/assets/4c2f0917-d873-4a86-acd0-c2996b483604">